numpy/core/shape_base.py

```diff
@@ -177,6 +160,82 @@ def atleast_3d(*arys):
         return res
 
 
+def atleast_nd(ary, n, pos='start'):
+    """
+    View input as array with at least `n` dimensions.
+
+    Parameters
+    ----------
+    ary : array_like
+        The input array. Non-array inputs are converted to arrays.
+        Arrays that already have `n` or more dimensions are preserved.
+    n : scalar
+        The minimum number of dimensions required for each of the input
+        arrays.
+    pos : {'start', 'end'}, optional
+        Where to add new dimensions. All new dimensions are added on the
+        same side of the shape. Default is 'start'.
+
+    Returns
+    -------
+    res : ndarray
+        An array with ``a.ndim >= n``. Copies are avoided where
+        possible, and a view with `n` or more dimensions is returned.
```

Comment:
I do not know enough about the internals to know if `res = array(ary, copy=False, subok=True)` will ever return a copy under normal circumstances. My guess is no, but I wanted to do some mild CYA in the docs, just in case.